### ISSUE TITLE
joystick not limited to "control' element

### ISSUE THREAD

**Issue by h3rb** [jeromeetienne/virtualjoystick.js#24]
I have an application that has a canvas that is 600x600 pixels, and I want the joystick only to function within that.

div id="control"
 canvas...
/div

id "control" is essentially the same as "container" in basic.html example, except it is set to 600px by  600px and contains the canvas.  However, On the left outside of this div I have elements that are not inside the control div yet the joystick appears so I cannot click those elements (sliders, text boxes, radio controls)

Yes, I have set container : document.getElementById('control')
Basically I need LimitStickActiveArea.html :)

**Comment by h3rb** [comment]
I tried wrapping the options I wanted to click in a div element and bringing it forward using zindex to no avail

**Comment by jeromeetienne** [comment]
See dual.html it contains a way to validate if a given NJ joystick position is valide or not

**Comment by h3rb** [comment]
I submitted a pull request, please review

**Comment by h3rb** [comment]
FYI dual.html shows me a blank page in Chrome

**Comment by jeromeetienne** [comment]
Try it on mobile

**Comment by jeromeetienne** [comment]
Or with touch event emulation in Chrome

**Comment by h3rb** [comment]
Well, that's not really a viable option right now so I added the functionality I needed and created a pull request.

**Comment by jeromeetienne** [comment]
i dont understand why you cant use that

**Comment by h3rb** [comment]
I'm not sure why it comes up with a blank page and no errors either.  I am not sure how to simulate touch environment, but in the end my additions, though they limited the values of x,y, did not solve the problem, what did solve the problem was removing the "relative" style option required for the container which is set by virtualkeyboard.js near line 14, after doing that (and turning off region checking features I added) things seem to work within the container only, which was desired.

**Comment by jeromeetienne** [comment]
well why not look harder. this feature is able to do exactly what you
descibed in the first email

**Comment by h3rb** [comment]
the issue is that it assigned it to be relative when i needed an absolute, perhaps a new feature could be changing the positioning of the resulting container, or leaving the css alone there and let the user set it in the style
